packages/map-template/README.md

```diff
@@ -1,88 +1,114 @@
-|:--|:--|:--|
-|`apiKey`|`string`|The MapsIndoors Solution you want to load. Takes both API key as string and "App alias". |
-|`gmApiKey`|`string`|Your Google Maps API key. |
-|`mapboxAccessToken`|`string`|Your Mapbox Access Token. Setting it will load a Mapbox map. If you set both a Mapbox Access Token and Google Maps API key, the Mapbox Access Token takes precedence. |
-|`venue`|`string`|The Venue to load from your MapsIndoors Solution. |
-|`locationId`|`string`|Set a MapsIndoors Location ID to show it on the map and its details in the sheet. |
-|`primaryColor`|`string`|The primary color to use throughout the app. |
-|`logo`|`string`|The logo to show during initial load. |
-|`appUserRoles`|`array`|A list of App User Roles to apply when loading data. Used like so: `appUserRoles={["App User Role"]}`|
-|`directionsFrom`|`string`|Set a MapsIndoors Location ID to be used as origin to instantly show directions. |
-|`directionsTo`|`string`|Set a MapsIndoors Location ID to be used as destination to instantly show directions. |
-|`externalIDs`|`array`|Array of external IDs which filters the map and shows a list of locations. Because of the way browsers work, you can not use External IDs with the `,`, `&`, `#` and `+`, character in them, as they are interpreted by the browser in a particular way. |
-|`tileStyle`|`string`|Name of Tile Style to display on the map. |
+Make sure the MapsIndoors JavaScript SDK is loaded by having this somewhere in your HTML:
 
-## Deploying Map Template to a cloud storage provider
+```html
+<script src="https://app.mapsindoors.com/mapsindoors/js/sdk/4.21.4/mapsindoors-4.21.4.js.gz"></script>
+```
 
-We often use Google Cloud Storage (GCS) for deploying small useful apps for demo purposes. This guide refers to GCS, but many of the steps are identical for AWS, Azure Blob, and the like.
+Use the `MapsIndoorsMap` component in a React component:
 
-Running the regular build command (`npm run build`), it's assumed that all links refer to the root of a domain. When you deploy to a storage bucket, you need to build the app with the bucket name preprended to all links. Vite has a build option to take care of this:
+```javascript
+import MapsIndoorsMap from '@mapspeople/map-template/dist/mapsindoors-react.es';
 
-```zsh
-$ vite build --base=/YOUR_BUCKET_NAME
+// Somewhere in your JSX:
+<div style={{
+      display: 'block',
+      width: '100vw',
+      height: '100vh'
+}}>
+      <MapsIndoorsMap></MapsIndoorsMap>
+</div>
 ```
 
-At this point you can upload the files manually to your bucket, or use the helpful CLI [`gsutil`](https://cloud.google.com/storage/docs/gsutil) for the purpose. This command uploads the complete `build` folder, and prevents the files from being cached:
-
-```zsh
-$ gsutil -m -h "Cache-Control:public, max-age=0, no-store, no-cache" cp -r build gs://YOUR_BUCKET_NAME
-```
+Add properties to the MapsIndoorsMap component as needed (see list below).
+
+# Supported attributes on the MapsIndoors Map components
```

Comment:
@ammapspeople More an observation than a change request, but I'm concerned about having this information across two READMEs. What are the chances of us forgetting to change one or the other? Can this be made in a different way? If not, this can work for now.

Reply:
It would be nice if markdown had some way to include other markdown files, but it doesn't seem so. Alternatively, we can just extract the table into a seperate markdown file and then link to it.

Reply:
I'm still a bit sad about the way this has to be, but alas.